src/pages/IncreaseLiquidity/index.tsx

```diff
@@ -697,8 +698,23 @@ export default function IncreaseLiquidity() {
                 </SecondColumn>
               </GridColumn>
             </AutoColumn>
-          ) : (
+          ) : loadingPosition || loadingInfo ? (
             <Loader />
+          ) : (
+            <Flex flexDirection="column" sx={{ gap: '16px' }}>
+              <Text fontSize="28px" fontWeight="500">
+                <Trans>404</Trans>
+              </Text>
+              <Text>
+                <Trans>There is no such position exists on {networkInfo.name}</Trans>
+              </Text>
+              <Text>
+                <Link to="#" onClick={toggleNetworkModal}>
+                  Switch network
+                </Link>{' '}
+                or <Link to={APP_PATHS.MY_POOLS}>go back to My Pools</Link>
+              </Text>
+            </Flex>
```

Comment:
Có lỗi đá nó ra my pool luôn đi k cần show gì đâu.

Reply:
it might causing flasing the page?